python/feathub/feature_views/transforms/tests/test_expression_transform.py

```diff
@@ -268,3 +268,74 @@ def _test_case(self, upper_name_feature: Feature):
         )
 
         self.assertTrue(expected_result_df.equals(result_df))
+
+    def test_duplicate_feature_name_with_source(self):
+        source = self.create_file_source(self.input_data.copy())
+
+        f_cost = Feature(name="cost", dtype=Int64, transform="cost + 1")
+
+        features = DerivedFeatureView(
+            name="feature_view",
+            source=source,
+            features=[
+                f_cost,
+            ],
+            keep_source_fields=True,
+        )
+
+        result_df = (
+            self.client.get_features(features)
+            .to_pandas()
+            .sort_values(by=["time"])
+            .reset_index(drop=True)
+        )
+
+        expected_result_df = self.input_data.copy()
+        expected_result_df["cost"] = expected_result_df.apply(
+            lambda row: row["cost"] + 1, axis=1
+        )
+        expected_result_df = expected_result_df.sort_values(by=["time"]).reset_index(
+            drop=True
+        )
+
+        self.assertTrue(expected_result_df.equals(result_df))
+
+    def test_refer_to_source_feature(self):
```

Comment:
It seems that this test includes the FeatHub program used in `test_duplicate_feature_name_with_source`. Would it be simpler to check the result of `feature_view_1` in this test and remove the other test?

The name `test_duplicate_feature_name_with_source` is probably better.